From 5bc08ed9b6187f73a9ea6d1b4d9d3e28aebeffaf Mon Sep 17 00:00:00 2001
From: Christian Lamparter <chunkeey@googlemail.com>
Date: Mon, 29 Apr 2013 21:33:17 +0200
Subject: [PATCH 61/65] r92su: update TODO
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

José Vázquez has tested the driver with:
  - TP-LINK TL-MR3220 v2. Wifi: AR9331 b/g/n OS: OpenWRT Driver: ath9k
  - Livebox 2. Wifi: AR9223 b/g/n OS: Linux 2.6.15 + OpenRG
    http://wiki.openwrt.org/toh/sagem/livebox.2
  - ARV4518PW. Wifi: AR2417 b/g OS: OpenWRT Driver: ath5k
    http://wiki.openwrt.org/toh/arcadyan/arv4518pw
  - Zyxel P660HW-D1. Wifi: Tnetw1350 b/g OS: ZynOS 3.x

Furthermore, I was able to confirm that the device
works with:
  - WNDR3700v2 Firmware 1.0.0.36
  - TP-Link TL-WR841ND v7 120201
  - AVM FritzBox! 7170
  - Alice SpeedBox (Not actually mine)

(As well as with several mac80211 aps)
  - ath9k_htc AP (WN721)
  - ath9k (R52N/AR9220, SR71-E/AR9280, ALL02456M/AR9380, N-1102/AR9382)
  - carl9170 AP (WNDA3100v1 and WN841N)
  - iwlagn (6200)
  - p54 (Allnet 0271, Dell 1450 USB)

Suspend and resume is also working.

throughput is the same as rtl8712u.
---
 TODO | 24 ------------------------
 1 file changed, 24 deletions(-)

diff --git a/TODO b/TODO
index 57a3b7f..2100cad 100644
--- a/TODO
+++ b/TODO
@@ -5,33 +5,9 @@ TODO:
    - That said, the firmware does not provide any counters?!
      So the statistics might be off and unuseable.
 
- - figure out which firmware commands are really implemented
-   in the firmware. Some just seem to be NOPs.
-   (Note: The same is true for firmware response. This will
-   be harder to figure out, as we need to trigger the condition
-   in the firmware somehow - and this is difficult)  
-
- - fix memory leaks
-
  - validate locking (especially RCU)
    spinlocks and mutex should be fine (tm).
 
-TESTS:
- - stability test
-   (How long does a idle/loaded connection last?)
-
- - performance/throughput test
-   (How fast is it, what's the jitter and package drop?)
-
- - interop test
-   (Can the driver/device connect to standard APs from different
-   vendors and how does it behave in a noisy environment)
-
- - suspend/resume cycle test
-   (Does a device automatically reconnect after a suspend/resume
-   cycle... Does the driver survive a cycle when the device was
-   briefly disconnected or has been removed)
-
 KNOWN LIMITATIONS:
  - No AP/P2P mode (firmware limitation)
  - No accurate tx feedback (fw-host protocol limitation)
-- 
1.8.3.1

